scripts/jenkins-support.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+function connect_rally_server_run_test
+{
+    # Connect to rally server and run tests
+    ssh -T root@$rally_server "bash -s $cloud $image_name" <<'EOF'
+cloud=$1
+image_name=$2
+
+source .openrc
+# Check if jeos image already exist on cloud
+openstack image show -c name --format value $image_name
+image_exist=$?
+# Upload jeos image
+if [[ $image_exist != 0 ]]; then
+    curl -s \
+    http://download.suse.de/ibs/SUSE:/SLE-12-SP2:/Update:/JeOS/images/SLES12-SP2-JeOS-for-OpenStack-Cloud.x86_64-1.2.0-Build13.308.qcow2 | \
+    openstack image create --public --disk-format qcow2 $image_name
+fi
+# Check if specific cloud deployemnt already exist
+rally deployment list | grep $cloud
+deployment_exist=$?
+if [[ $deployment_exist != 0 ]]; then
```

Comment:
This strikes me as strange -- if you want the exit code of the grep, we probably want to set -o pipefail.

Maybe we can avoid that with a subshell?

if ! (rally deployment list | grep $cloud); then